src/test/scala/org/zalando/kanadi/SubscriptionsSpec.scala

```diff
@@ -70,14 +57,17 @@ class SubscriptionsSpec(implicit ec: ExecutionEnv) extends Specification with Co
       subscription
     })
 
-    val retrievedSubscriptions = (for {
+    val retrievedSubscriptions = for {
       subscriptions <- createdSubscriptions
-      retrievedSubscription = Future.sequence(subscriptions.map { subscription =>
-                                subscriptionsClient.createIfDoesntExist(subscription)
-                              })
-    } yield retrievedSubscription).flatMap(a => a)
+      retrievedSubscription <- Future.sequence(subscriptions.map { subscription =>
```

Comment:
Not sure why left arrow wasn't being used here, `Future.sequence` also returns a future so we should yield on it like another other `Future`.